e2e/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/cosmos/ibc-go/e2e
 
 go 1.21
 
+replace github.com/strangelove-ventures/interchaintest/v8 => github.com/DimitrisJim/interchaintest/v8 v8.0.0-20240419095404-2c9270423b9a
```

Comment:
Is the fork necessary or is there a commit on interchaintest which we can point at?

Would be good to add a comment either way for what the diff is and/or when we can remove it